src/verifiers/issuerIdentity/did/didIdentityProof.ts

```diff
@@ -112,7 +111,7 @@ const verifyV3 = async (document: v3.WrappedDocument, options: VerifierOptions):
 const verify: VerifierType["verify"] = withCodedErrorHandler(
   async (document, options) => {
     if (utils.isWrappedV2Document(document)) return verifyV2(document, options);
-    if (utils.isWrappedV3Document(document)) return verifyV3(document, options);
+    else if (utils.isWrappedV3Document(document)) return verifyV3(document, options);
```

Comment:
Just curious how come need to add the else since there is a return statement in the condition which act as a break before the next condition check?

Reply:
imho it's more readable (can argue) but more importantly it's more logical., whether there is a return or not. I don't want to change the if/else structure depending on whether thee branch returns or not